src/clusterfuzz/_internal/bot/fuzzers/libfuzzer.py

```diff
@@ -428,6 +429,24 @@ def _clear_all_target_corpora(self):
     # prepare_fuzzer resets the data/ directory
     undercoat.prepare_fuzzer(self.handle, self.executable_path)
 
+  def _ensure_target_exists(self):
+    """ Check that the target fuzzer exists, raising an error if it does not.
```

Comment:
nit: extra space at the beginning.

Reply:
Oops, thanks :) I was mirroring the format of the functions above, so I've clearly done this before-- will try to remember next time!